test/ModalSpec.js

```diff
@@ -353,8 +371,7 @@ describe('Modal', function () {
     });
 
     it('Should focus on the Modal when it is opened', function () {
-
-      document.activeElement.should.equal(focusableContainer);
+      expect(document.activeElement).to.equal(focusableContainer);
```

Comment:
Does that make a difference?

Reply:
it seems to locally. I'm not at all sure why

Reply:
could be flakiness with the prototype manipulation stuff